common/state_traits/07_africa_traits.txt

```diff
@@ -59,14 +64,17 @@ state_trait_zambezi_river = {
 	
 	modifier = {
 		state_infrastructure_add = 25
+		building_group_bg_agriculture_throughput_add = 0.15
+		building_group_bg_plantations_throughput_add = 0.15
+		building_group_bg_fishing_throughput_add = 0.10
```

Comment:
No MAPI?

Reply:
Apparently it wasnt very navigable because of the flooding/dry seasons, but it did make the land very rich in soil and fish

Reply:
Ah, I see! Approved then :)